### ISSUE TITLE
Savestates incorrectly loading into Black Screen [Unhandle error/ Black screen + Audio]

### ISSUE THREAD

**Issue by CorpseSlayer** [RPCS3/rpcs3#14717]
### Quick summary

Before anything i want to make sure  to try to explain the best i can with this new Issue
for some reason savestate seems to be working the problem is when loading the savestate
it either gives Unhandle error after loading the savestate or Black screen with Audio only after 
loading the savestate depends on what game you are playing for me the unhandle error appears
while playing Castlevania lord of shadows, but in Dante's Inferno savestate saves correctly but when 
loading it , it gives black  screen with audio so some is wrong with savestate another thing i want it
to mention is that savestate are being made throw the RPCS3 menu.

![273031060-ab5503aa-5b85-4d11-9e0e-f536625393a6](https://github.com/RPCS3/rpcs3/assets/126618918/bb2cb722-ecf5-4962-beee-829a3a1aee98)

Advance compatibility option is Enable = Enable

![273030885-e5d0895a-e8f5-4d1b-8496-1842f2c6cd13](https://github.com/RPCS3/rpcs3/assets/126618918/61aa6c6b-6e5c-4988-9e87-3c1c596fcfad)

### Details

RPCS3 is loading Savestates incomplete some with Black screen with audio [Dante's Inferno] 
Others Savestate loads with black screen with and unhandled error [Castlevania lord of shadows]

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/12836170/RPCS3.log)
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12836171/RPCS3.log.gz)
[TTY.log](https://github.com/RPCS3/rpcs3/files/12836172/TTY.log)

### System configuration

Windows 11
16 Gb ram
Nvidia RTX 2060

**Comment by elad335** [comment]
Retry on latest master

**Comment by CorpseSlayer** [comment]
Working good, I launch the game and play for a while and use the menu to savestate at first i thought it freeze but it just takes a while to complete the savestate then it reopen the game and it load the game just fine no blackscreen + music is working, i fully close the game and open it choosing load with savestate and it load just fine no problems. thank you for fixing this issue.

**Comment by CorpseSlayer** [comment]
> Retry on latest master
 Working good, I launch the game and play for a while and use the menu to savestate at first i thought it freeze but it just takes a while to complete the savestate then it reopen the game and it load the game just fine no blackscreen + music is working, i fully close the game and open it choosing load with savestate and it load just fine no problems. thank you for fixing this issue.
